package/MDAnalysis/analysis/encore/clustering/ClusteringMethod.py

```diff
@@ -195,6 +195,14 @@ def __init__(self,
                 (default is 50). Parameter in the Affinity Propagation for
                 clustering.
 
+            random_state : int, RandomState instance or None, optional
+		Used to control the random number generator used, for the
+		randomization which is part of scikit-learn algorithm(default
+		is scikit-learn's default).Use an int(unique) for reproducible
+		results across function calls. Previously(version<0.23), this
+		parameter was hardcoded to 0 by scikit-learn. Refer its
+		documentation for more details.
+
```

Comment:
Please don't mix tabs and spaces. This is detailed documentation, but I'm not sure it's necessary to be documented -- you could just document `kwargs` as being sent to scikit.

Reply:
Done! I will take care regarding this from next time :)